htmltools/_core.py

```diff
@@ -392,14 +409,19 @@ class Tag:
     """
     The HTML tag class.
 
+    A Tag object consists of a name, attributes, and children. The name is a string, the
+    attributes are held in a TagAttrDict object, and the children are held in a TagList
+    object.
+
+    This class usually should not be instantiated directly. Instead, use the tag wrapper
+    functions in ``htmltools.tags``, like ``div()`` or ``a()``.
```

Comment:
I think technically this should use RST syntax to link to the class/func/type, like this ...

https://github.com/rstudio/py-shiny/blob/d5855b0189d1d2385bf622f73edd3bb68bf8f125/shiny/ui/_input_date.py#L45

... but I doubt we're doing that a good job of that in this repo, so I feel free to ignore :)

Reply:
Good catch. We're going to do a pass over this repo fixing docs once we get quartodoc working properly, so I'll just save it for then.